portal.go

```diff
@@ -2181,6 +2182,21 @@ func (portal *Portal) HandleWhatsAppInvite(source *User, senderJID *types.JID, j
 	return
 }
 
+func (portal *Portal) HandleWhatsAppDeleteChat() {
+	matrixMembers, err := portal.GetMatrixUsers()
+	if err != nil {
+		portal.log.Errorln("Couldn't get Matrix users to figure out if deleteChat should be handled!")
+		return
+	}
+	if len(matrixMembers) > 1 {
```

Comment:
This should check that the member is the user who deleted the chat just to be safe (like deleteForMe does)